project/Versions.scala

```diff
@@ -26,7 +26,7 @@ object PagopaVersions {
   lazy val purposeProcessVersion              = "1.0.x-SNAPSHOT"
   lazy val tenantManagementVersion            = "1.0.x-SNAPSHOT"
   lazy val tenantProcessVersion               = "1.0.x-SNAPSHOT"
-  lazy val partyManagementClientVersion       = "1.0.x-SNAPSHOT"
+  lazy val partyManagementClientVersion       = "1.0.11"
   lazy val certifiedMailSenderModelsVersion   = "1.0.x-SNAPSHOT"
-  lazy val selfcareV2ClientVersion            = "1.0.x-SNAPSHOT"
+  lazy val selfcareV2ClientVersion            = "1.0.11"
```

Comment:
This must be fixed until https://github.com/pagopa/interop-be-jobs/pull/158 is not merged.